packages/migrations/src/batched-migrations/batched-migration-runner.ts

```diff
@@ -177,6 +177,10 @@ export class BatchedMigrationRunner {
       }
 
       if (error) {
+        logger.error(
+          'Error running the batched migration job. To re-run the migration job, open the Admin Panel and click "Retry Failed Jobs".',
```

Comment:
We don't show recovery instructions for other kinds of errors, is it necessary to do so here?

Reply:
I'm not sure. For non-batch migrations, an error in the migration causes it to re-run the next time you restart the server. A batched migration does not do that, you need to re-run explicitly. I don't mind removing the instructions if you believe they shouldn't be there, but given this difference in behaviour it may be interesting to keep this in mind.

Reply:
The way I'm thinking about this, simply re-running it won't magically make it work in most cases; one would have to make a change to the migration code itself. I also don't expect folks to run into this during local dev much (we shouldn't be landing broken migrations to master), and if someone does run into it, it's likely they're actively working on batched migrations and would know to re-run it.

In production environments, printing "Click this button" over and over again, especially in the _middle_ of execution (when one probably wouldn't want to rerun them without changes) seems undesirable.

Reply:
Makes sense. Batched migrations are something relatively new, so this may become a bit easier to handle once its more commonplace. Removed in e278ae2cfb1382649742393c2a602682e138d1dc.